index.js

```diff
@@ -0,0 +1,36 @@
+const item = {
+  "itemId": '02ccbfc7-e12d-4153-abd8-b5745buja6b5',
+  name: 'Bulldog Tie',
+  description: 'Impress all of your friends and associates with this Drake Bulldog Tie!',
+  price: 19.99,
+  sizesAvailable: ' L, M, S'
+};
+
+const firstName = 'Bill';
+const lastName = 'Nye';
+
+const customer = {
+customerId: 'qohifeu8-1dds9-41c8-3f0i-53dsknj1eq3',
+firstName,
+lastName,
+email: `${firstName.toLowerCase()}.${lastName.toLowerCase()}@drake.edu`,
+phoneNumber: '+15155544455'
+};
+
+const cart = {
+cartId: 'biwsdv8-1sv8-46v9-7m0q-53fbd56voe2',
+customerId: 'qohifeu8-1dds9-41c8-3f0i-53dsknj1eq3',
+createdDate: '2020-02-07',
+purchasedDate: '2020-02-10'
+};
+
+const cartItem = {
```

Comment:
```suggestion

const CartItems= [
   {
   cartid: 'c573f256-50d7-11ea-8d77-2e728ce88125',
   cartItemId: '4b6c8cfe-512a-11ea-8d77-2e728ce88125',
   itemid:'b7671ac6-50d7-11ea-8d77-2e728ce88125',
   quantity: 3
   },
   {
   cartid: 'c573f256-50d7-11ea-8d77-2e728ce88125',
   cartItemId: '5b6c8cfe-512a-11ea-9d66-2e728ce88135',
  itemid:'c7761ac6-40d7-11eb-8e77-2d728ce88124',
   quantity: 2
   }
  ];

```

cart items will be a list and each cart item will be part of a single cart so they will share the same cartId